public/app/features/explore/QueryLibrary/QueryTemplatesList.tsx

```diff
@@ -40,6 +43,25 @@ export function QueryTemplatesList(props: QueryTemplatesListProps) {
         return;
       }
 
+      let userDataList;
+      try {
+        const userQtList = uniq(data.map((qt) => qt.user?.uid));
+        const usersParam = userQtList.map((user) => `key=${user}`).join('&');
+        userDataList = await getUserInfo(`?${usersParam}`);
+        setUserData(userDataList.display.map((user) => user.displayName));
+      } catch (error) {
```

Comment:
If we are catching just the request error we should wrap only that. Probably not high chance but for example I don't know if `setUserData` could potentially error out if it creates a cyclic updates in react which would then be reported `Error attempting to get user info from the library` which isn't really true and may mask the error a bit.

Reply:
That makes sense. I  forgot to remove this logic out of try block. I've updated it :)